content/en/platform/corda/4.11/enterprise/network/available-network-parameters.md

```diff
@@ -11,46 +11,100 @@ title: Available network parameters
 in the compatibility zone.
 
-* **maxMessageSize**:
-Maximum allowed size in bytes of an individual message sent over the wire.
+## `recoveryMaximumBackupInterval`
 
-* **maxTransactionSize**:
-Maximum allowed size in bytes of a transaction. This is the size of the transaction object and its attachments.
+The `recoveryMaximumBackupInterval` network parameter is an optional parameter of type Duration, and is used by [Ledger Recovery](ledger-recovery.md). It specifies how far back in time the recovery process should consider. When attempting a recovery, a node will only restore to a database backup more recent than this value.
 
-* **modifiedTime**:
-The time when the network parameters were last modified by the compatibility zone operator.
+This value can be overridden by specifying an override in the flow. It can also be overridden for a particular node if the same parameter is specified in the node configuration; the node configuration takes precedence over the network configuration. An override to the flow takes priority over values in either the network configuration or node configuration. 
 
-* **epoch**:
-Version number of the network parameters. Starting from 1, this will always increment whenever any of the
-parameters change.
+The parameter is optional in both the network parameters and the node configuration. However, if no values are set then it needs to be specified in the flow.
 
-* **whitelistedContractImplementations**:
-List of whitelisted versions of contract code.
-For each contract class there is a list of SHA-256 hashes of the approved CorDapp jar versions containing that contract.
-Read more about *Zone constraints* here api-contract-constraints
+For example:
 
-* **eventHorizon**:
-Time after which nodes are considered to be unresponsive and removed from network map. Nodes republish their
-`NodeInfo` on a regular interval. Network map treats that as a heartbeat from the node.
+```
+recoveryMaximumBackupInterval = 10m
+```
 
-* **packageOwnership**:
-List of the network-wide java packages that were successfully claimed by their owners.
+## `packageOwnership`
+
+The list of the network-wide Java packages that were successfully claimed by their owners.
 Any CorDapp JAR that offers contracts and states in any of these packages must be signed by the owner.
-This ensures that when a node encounters an owned contract it can uniquely identify it and knows that all other nodes can do the same.
-Encountering an owned contract in a JAR that is not signed by the rightful owner is most likely a sign of malicious behaviour, and should be reported.
+This ensures that when a node encounters an owned contract, it can uniquely identify it and knows that all other nodes can do the same.
+Encountering an owned contract in a JAR that is not signed by the rightful owner is most likely a sign of malicious behavior, and should be reported.

```

Comment:
I assume the American spelling is intentional here

Reply:
Yes, our docs should be in US English....